recipes/hpyx/recipe.yaml

```diff
@@ -0,0 +1,85 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+schema_version: 1
+
+context:
+  name: hpyx
+  version: "2025.8.28"
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/${{ name }}-${{ version }}.tar.gz
+  sha256: ce582d60d0ad50256fc8f6666d5c30be9f15af8936a8711c5f2f03404caf2958
+
+build:
+  script:
+    - if: osx
+      then: |
+        export CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
+    - python -m pip install . --no-deps --ignore-installed -vvv
+  number: 0
+  python:
+    # Option to specify whether a package is version independent (aka ABI3)
+    version_independent: true  # defaults to fals
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compiler names include 'c', 'cxx' and 'fortran', among others.
+    - ${{ compiler('cxx') }}
+    - ${{ stdlib("c") }}
+  host:
+    - python 3.13.* *_cp313t
+    - pip
+    - scikit-build-core
+    - setuptools_scm
+    - cmake >=3.15,<3.29
+    - make
+    - if: win
+      then: git
+    - if: not win
+      then: ninja
+    - hpx
+    - asio
+    - libboost-devel
+    - if: not win
+      then: gperftools
+    - libhwloc ==2.12.1
+    - if: win
+      then: mimalloc
+    - nanobind
+  run:
+    - python 3.13.* *_cp313t
+    - numpy
+
+tests:
+  - package_contents:
+     site_packages:
+      - hpyx
+  - python:
+      imports:
+      - hpyx
+      pip_check: true
+  - script:
+      - if: linux
+        then: |
+          export LD_PRELOAD="$PREFIX/lib/libtcmalloc_minimal.so:$LD_PRELOAD"
```

Comment:
Why is this needed?

Reply:
This is because of a know issue when using tcmalloc from the dependency low level library `hpx`. Adding `LD_PRELOAD` fixes the issue, this is similar to https://github.com/STEllAR-GROUP/phylanx/wiki/Phylanx-Issues-and-Answers#phylanx-crashes-python. @hkaiser can explain this better to the detail of why this is potentially happening, if you need a more detailed answer. Thanks!

Reply:
Using LD_PRELOAD should not be used in conda-forge packages. It's a bad pattern and bound to lead to errors. More details please.